libbeat/docs/processors-using.asciidoc

```diff
@@ -748,3 +749,33 @@ The fields added to the event are looking as following:
 -------------------------------------------------------------------------------
 
 NOTE: The host information is refreshed every 5 minutes.
+
+[[dissect]]
+=== Dissect
+
+The `dissect` processor allows you to use the `dissect` syntax to tokenizer incoming string into
+defined patterns.
+
+
+Note that if the key exists in the `target_field`, the original value will be replaced.
```

Comment:
I suggest we don't allow this and someone would have to use drop_field before. This would be the same behaviour as we have in the rename processor.

Reply:
I am OK with that.